website/citations/views.py

```diff
@@ -13,18 +12,16 @@
 )
 
 def list_citation_styles():
-    query = None
-
-    term = request.args.get('q')
-    if term:
-        query = (
-            Q('_id', 'icontains', term) |
-            Q('title', 'icontains', term) |
-            Q('short_title', 'icontains', term)
+    query = request.args.get('q')
+    citation_styles = CitationStyle.objects.all()
```

Comment:
The query won't be executed until `citation_styles` is evaluated, so there's actually no duplication here. 

https://docs.djangoproject.com/en/1.11/ref/models/querysets/#when-querysets-are-evaluated